dist/taginfo.json

```diff
@@ -11,12 +11,14 @@
     "contact_email": "bryan@mapbox.com"
   },
   "tags": [
+    {"key": "access", "value": "customers"},
```

Comment:
ugh, how did this sneak in 😕

Reply:
https://github.com/osmlab/name-suggestion-index/blob/94f7cf527f4570efb85f6556f3223ccc6700e2ae/brands/leisure/playground.json - hard to say is it correct, McDonalds in Poland sometimes have playgrounds but not branded ones

Reply:
I took them out in 2782d7b6